ext/opentelemetry-ext-botocore/setup.cfg

```diff
@@ -43,6 +43,7 @@ install_requires =
     botocore ~= 1.0
     opentelemetry-api == 0.11.dev0
     opentelemetry-instrumentation == 0.11.dev0
+    opentelemetry-ext-boto == 0.11.dev0
```

Comment:
I'm not too familiar with boto or botocore. Should the dependency be the other way around? Is botocore the more `lower-level` integration?

Reply:
Fixed